internal/provider/resource_consul_cluster.go

```diff
@@ -129,6 +128,12 @@ func resourceConsulCluster() *schema.Resource {
 					return strings.ToLower(old) == strings.ToLower(new)
 				},
 			},
+			"auto_hvn_to_hvn_peering": {
```

Comment:
re 2) we should default to `false` as discussed with the platform-team.
re 3) personally I think we should merge this PR whenever it is ready. There is a LD flag for it that is being checked and I could make that always return true. Right now it is only enabled in dev env.

Reply:
re 2/4. If the API sets the default value, you should add the line `Computed: true` to indicate the value will be set during Create even if the user doesn't provide a value.

re 3. While I don't foresee this happening, it is always possible we might need to cut an emergency release of the provider to fix a bug. So if this merged before the HVN peering resource was done, and then got pulled into a bug fix, it could be awkward. So maybe worth holding off merging until HVN peering is in just to be safe.

Reply:
@bcmdarroch Do I need to set `Computed: true` when terraform and the API uses the default `false`?

Reply:
Yes, I think so, if only to be explicit that it's set by the API.